### PR TITLE
Feature/skale 2725 extend validator commands

### PR DESCRIPTION
**Pull request by dmytrotkk**

### CODE REVIEW

File: core/validator.py [hunk 1]
```diff
@@ -177,3 +177,53 @@ def get_bond_amount(validator_id, wei=False):
...
+        tx_res = skale.validator_service.request_for_new_address(
+            new_validator_address=address,
+            raise_for_status=False
+        )
+        if not check_tx_result(tx_res.tx_hash, skale.web3):
+            sp.write(f'Transaction failed, hash: {tx_res.tx_hash}')
+            return
+        sp.write(
+            f'✔ Requested new address for your validator ID: {address}.\n'
+            'You can finish the procedure by running < skale validator confirm-address > '
```
Comment (cstrangedk) [inline core/validator.py]: command should be `sk-val validator confirm-address [VALIDATOR ID]`

File: tests/cli/validator_test.py [hunk 2]
```diff
@@ -288,3 +300,60 @@ def test_bond_amount(runner, skale):
...
+        [
+            wallet.address,
+            '--pk-file', TEST_PK_FILE,
+            '--yes'
+        ]
+    )
+    output_list = result.output.splitlines()
+    assert result.exit_code == 0
+    assert f'\x1b[K✔ Requested new address for your validator ID: {wallet.address}.' in output_list
+    assert f'You can finish the procedure by running < skale validator confirm-address > using the new key.' in output_list  # noqa
```
Comment (cstrangedk) [inline tests/cli/validator_test.py]: update assert

Comment (dmytrotkk) [reply tests/cli/validator_test.py]: @cstrangedk addressed

**Review by badrogger — APPROVED** [review]
Great!

**Pull request merged**